### ISSUE TITLE
Rheda: Updating replacement player to nobody not working.

### ISSUE THREAD

**Issue by Jesterboxboy** [MahjongPantheon/pantheon#258]
Removing a replacement player via Rheda(on view http://localhost:4002/eid2/reg) does not work.
Field in db is not updated.
Had the problem at the last tournament, because a player showed up in the last minute and i needed to remove the replacement player again, which i then did directly from the event_registered_player table.

**Comment by Jesterboxboy** [comment]
Removing a replacement player via Rheda(on view http://localhost:4002/eid2/reg) does not work.
Field in db is not updated.

**Comment by ctizen** [comment]
If there are any other thoughts about this issue from other team members, please share:)

**Comment by Jesterboxboy** [comment]
After testing a bit i think it's just a frontend bug,
Selecting "Nobody" in Rheda in eid2/reg in the replacement player field, does not trigger a backend api call to mimir.
I was able to fix it to some degree, so at least after reloading the page, clicking "Nobody" triggers a api call reliably, but not after populating the whole player list in the dropdown. There seems to be some problem with the string concatenation in the _updateReplacementList function in the PlayerRegistration.handlebar template file.
I commited the changes to my fork if you wanna have a look:
https://github.com/Jesterboxboy/pantheon/commit/9a01da88a47588d0990aeb39a6334ff62c1ce5b9
